javascript-data-model/order-history.js

```diff
@@ -0,0 +1,64 @@
+var orderHistory = [
+  order1 = [
+    {
+      orderPlaced: 'August 4, 2020',
+      total: '$34',
+      shipTo: 'JS Masher',
+      order: '114-3941689-877232',
+      deliveredDate: 'Aug 8, 2020',
+      bookTitle: 'Javscript for Impantient programmers',
+      bookAuthor: 'Rauschmayer, Dr.Axel',
+      returnWindowClosingDate: 'Sep 7, 2020',
+      price: '$31.55'
+    }
+  ],
+  order2 = [
+    {
+      orderPlaced: 'July 19,2020',
+      total: '$44.53',
+      shipTo: 'JS Masher',
+      order: '113-9984268-1280257',
+      deliveredDate: 'Jul 20,2020',
+      bookTitle: 'The Timeless Way of Building',
+      bookAuthor: 'Alexander, Christopher',
+      returnWindowClosingDate: 'Aug 19 2020',
+      price: '$41.33'
+    }
+  ],
+  order3 = [
+    {
+      orderPlaced: 'July 4,2020',
+      total: '$17.22',
+      shipTo: 'JS Masher',
+      order: '114-2875557-9059049',
+      deliveredDate: 'Jul 7,2020',
+      itemName: 'Gamecube Controller Adapter, Super Smash Bros Switch Gamcube Adapter for WII U,PC Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      returnWindowClosingDate: 'Aug 5, 2020',
+      price: '$15.98'
+    }
+  ],
+  order4 = [
+    {
+      orderPlaced: 'July 3,2020',
+      total: '$138.93',
+      shipTo: 'JS Masher'
+    },
+    {
+      order: '113-2883177-2648248',
+      delivereDate: 'July 3,2020',
+      itemName: 'GameCube Controller - Super Smash Bros Edition (Nintendo Switch)',
+      returnWindowClosingDate: 'Aug 4,2020',
+      price: '$94.94'
+    },
+    {
+      order: '113-2883177-2648248',
+      deliveredDate: 'July 3, 2020',
+      bookTitle: 'The Art of Sql',
+      bookAuthor: 'Faroult Stephane',
+      returnWindowClosingDate: 'Aug 4,2020',
+      price: '$33.99'
+
+    }
+  ]
```

Comment:
Combine all this into one object with a similar structure to this:

```
{
    orderDate: 20200703,
    shipTo: 'JS Masher',
    orderNumber: 11328831772648248,
    items:
    [
      {
        name: 'Gamecube Controller - Super Smash Bros. Edition (Nintendo Switch)',
        price: 94.95
      },
      {
        deliveryDate: 20200705,
        name: 'The Art of Sql',
        price: 33.99
      }
    ]
  }
```

Reply:
![Screen Shot 2021-01-28 at 3 18 35 PM](https://user-images.githubusercontent.com/69738204/106194178-7f07f900-617c-11eb-871a-9060f5362a84.png)
![Screen Shot 2021-01-28 at 3 18 44 PM](https://user-images.githubusercontent.com/69738204/106194184-80d1bc80-617c-11eb-9953-69633b048f7a.png)